linkedin2username.py

```diff
@@ -122,6 +122,9 @@ def parse_arguments():
                         help='Attempts to bypass the 1,000 record search limit'
                         ' by running multiple searches split across geographic'
                         ' regions.')
+    parser.add_argument('-o', '--output', default=False, action="store", required=True,
+                        type=str, help='Output Directory')
```

Comment:
```suggestion
    parser.add_argument('-o', '--output', default="li2u-output", action="store",
                        type=str, help='Output directory.')
```

This will allow the tool to continue functioning as-is, without requiring the user to specify an output directory.

Reply:
awesome changes made!